groups/templates/groups/group_edit.html

```diff
@@ -1,4 +1,7 @@
 {% include 'base.html' %}
+{% block content %}
+{% load tz %}
+{% timezone "US/Pacific" %}
```

Comment:
it is changed to US/Pacific at server side for now. Above code is removed actually...